crates/re_types/definitions/rerun/blueprint/archetypes/panel_blueprint.fbs

```diff
@@ -0,0 +1,21 @@
+include "fbs/attributes.fbs";
+include "rust/attributes.fbs";
+
+namespace rerun.blueprint.archetypes;
+
+// ---
+
+/// Shared state for the 3 collapsible panels.
+table PanelBlueprint (
```

Comment:
I'm not sure if that is the right split. Realistically we'll want to have different archetypes for time panel, blueprint panel and selection panel each. All of them then have an `expanded` property. Splitting these panel internally in several archetypes might come up, but it would be great if we can avoid it I'd day.

Reply:
Discussed in person. Leaving as is for now but introduced per-panel helpers in api.py.